lambda/delete-activity-log/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "delete-activity-log",
+  "version": "1.0.0",
+  "description": "A Lambda function to delete activity log records when the account is deleted",
+  "main": "delete-activity-log.js",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint . --ext .ts",
+    "test:cov": "jest --coverage"
+  },
+  "author": "GDS",
+  "license": "OGL",
+  "devDependencies": {
+    "@babel/core": "^7.19.3",
+    "@babel/preset-env": "^7.19.3",
+    "@babel/preset-typescript": "^7.18.6",
+    "@types/aws-lambda": "^8.10.106",
+    "@types/jest": "^29.5.2",
+    "@typescript-eslint/eslint-plugin": "^5.40.0",
+    "aws-sdk-client-mock": "^2.0.0",
+    "aws-sdk-client-mock-jest": "^2.0.0",
+    "aws-sdk-mock": "^5.8.0",
+    "babel-jest": "^29.1.2",
+    "eslint": "^8.25.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-config-airbnb-typescript-prettier": "^5.0.0",
+    "eslint-import-resolver-typescript": "^3.5.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "jest": "^29.1.2",
+    "prettier": "^2.7.1",
+    "ts-jest": "^29.0.3",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4"
+  },
+  "dependencies": {
+    "@aws-sdk/client-sqs": "^3.186.0",
```

Comment:
There is a vulnerability that is pulled as a part of this version of client-sqs https://github.com/alphagov/di-account-management-backend/pull/112/files